src/wallet/abstract-hd-electrum-wallet.js

```diff
@@ -1048,4 +1048,32 @@ export class AbstractHDElectrumWallet extends AbstractHDWallet {
     }
     return false;
   }
+
+  /**
+   * @param seed {Buffer} Buffer object with seed
+   * @returns {string} Hex string of fingerprint derived from mnemonics. Always has lenght of 8 chars and correct leading zeroes. All caps
+   */
+  static seedToFingerprint(seed) {
+    const root = HDNode.fromSeed(seed);
+    let hex = root.fingerprint.toString('hex');
+    while (hex.length < 8) hex = '0' + hex; // leading zeroes
+    return hex.toUpperCase();
+  }
+
+  /**
+   * @param mnemonic {string}  Mnemonic phrase (12 or 24 words)
+   * @returns {string} Hex fingerprint
+   */
+  static mnemonicToFingerprint(mnemonic) {
+    const seed = bip39.mnemonicToSeedSync(mnemonic);
+    return AbstractHDElectrumWallet.seedToFingerprint(seed);
+  }
+
+  /**
+   * @returns {string} Hex string of fingerprint derived from wallet mnemonics. Always has lenght of 8 chars and correct leading zeroes
+   */
+  getMasterFingerprintHex() {
+    const seed = this._getSeed();
+    return AbstractHDElectrumWallet.seedToFingerprint(seed);
+  }
```

Comment:
Also curious if these are covered by unit tests?

Reply:
getMasterFingerPrint is not required so have removed it in a new commit.